app/src/androidTest/java/com/duckduckgo/app/browser/BrowserTabViewModelTest.kt

```diff
@@ -3012,13 +3009,24 @@ class BrowserTabViewModelTest {
     }
 
     private inline fun <reified T : Command> assertCommandNotIssued() {
-        val issuedCommand = commandCaptor.allValues.find { it is T }
-        assertNull(issuedCommand)
+        val defaultMockingDetails = DefaultMockingDetails(mockCommandObserver)
+        if (defaultMockingDetails.invocations.isNotEmpty()) {
+            verify(mockCommandObserver, atLeastOnce()).onChanged(commandCaptor.capture())
+            val issuedCommand = commandCaptor.allValues.find { it is T }
+            assertNull(issuedCommand)
+        } else {
+            assertTrue(true)
```

Comment:
Not doing a full review as I haven't tested, but just skimming it

`assertTrue(true)` seems unnecessary. We only want to assert when there's a chance of the assert failing; `assertTrue(true)` will always pass, so you're as well doing nothing at all.

Reply:
Good point, I will remove it.